### ISSUE TITLE
Tree Control - Changed Selection Look

### ISSUE THREAD

**Issue by zanderson9** [vincenzopalazzo/material-ui-swing#127]
in 1.1.1_0.1 - JTree nodes highlight with the already established theme has worse visibility (white on gold) than in version prior to 1.1.1_0.1

before 1.1.1_0.1 - looks good

![Tree-Good](https://user-images.githubusercontent.com/49419742/77603673-eb1b3000-6ecd-11ea-8e12-4664ca302382.JPG)

with 1.1.1._0.1  - visibility is worse - white on gold not too visible to many users...

![Tree-bad](https://user-images.githubusercontent.com/49419742/77603704-0128f080-6ece-11ea-86d0-73d3aef79bc0.JPG)

what settings need to change in the custom theme to have just gold highlight?

**Comment by vincenzopalazzo** [comment]
Hi @zanderson9,

Inside your configuration, you should insert the 


```
        UIManager.put("Tree.selectionBackground", COLOR);
        UIManager.put("Tree.selectionBorderColor", COLOR);
```

**Comment by zanderson9** [comment]
Those 2 additional settings did resolve the issue.
Thank you.

**Comment by vincenzopalazzo** [comment]
Fixed inside material-ui-swing-1.1.1-rc1
